### PR TITLE
Fix Process

### PR DESCRIPTION
**Pull request by MarwanElkhoudary**
## Fix Process

- [x]  Fix authentication 
- [x]  Fix signup page
- [x]  Fix login Page
- [x] Fix logout
- [x] Fix proteced Pages.
- [x] Fix transition process between pages.

### CODE REVIEW

**Comment by maxgerber** [comment]
Can you relate this PR to the issues it will fix?
i.e. "Fixes #..., #..."

**Comment by MarwanElkhoudary** [comment]
> Can you relate this PR to the issues it will fix?
> i.e. "Fixes #..., #..."

File: client/src/Components/App.js [hunk 1]
```diff
@@ -8,111 +9,121 @@ import workExperienceDetails from './Pages/WorkExpDetails';
...
+              measure="%"
+            >
+              {this.getModal(CurrentModal)}
+            </Rodal>
+
+            <div className="app-content">
+              <Switch>
+                <Route exact path="/" component={LandingPage} />
+                <Route path="/dash" component={Dashboard} />
+                {/* <Route path="/signup" component={Signup}/> */}
```
Comment (hshahwan) [inline client/src/Components/App.js]: delete the comment lines

File: client/src/Components/Pages/Dashboard/PersonalInfo/index.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
-import React, {
-  Component,
-} from 'react';
+import React, { Component } from 'react';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import axios from 'axios';
 // import propTypes from 'prop-types';
```
Comment (hshahwan) [inline client/src/Components/Pages/Dashboard/PersonalInfo/index.js]: delete this line too

File: client/src/Components/Pages/WorkExpList/index.js [hunk 3]
```diff
@@ -9,20 +9,27 @@ class workExperienceList extends Component {
   };
 
   componentDidMount() {
+   if (document.cookie) {
     const { action } = this.props.history;
+    console.log('jjjjjj', action);
```
Comment (hshahwan) [inline client/src/Components/Pages/WorkExpList/index.js]: delete the console logs

File: server/controllers/school.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ const { school } = require('../models')
 const sequelize = require('../models/connection');
 
 const getSchoolsList = async (request, response) =>{
+    console.log('reqqqqqqqqqqqqqqq',request);
```
Comment (hshahwan) [inline server/controllers/school.js]: delete this line

**Pull request merged**